golang/UPDATING.md

```diff
@@ -4,32 +4,21 @@
 
 #### One-Time Setup
 
-1. Download and setup Protocol Buffer release from https://github.com/protocolbuffers/protobuf/releases (if you haven't already done this for another language).  As of February 2019 we're using v3.7 release, which is compatible with proto2 .proto files.
-1. Download [Go](https://golang.org/dl/)
+1. Download and install [Docker](https://docs.docker.com/get-docker/)
 
 #### Every time `gtfs-realtime.proto` changes
 
-1. Regenerate the language binding source from gtfs-realtime.proto by running the following from the `golang` directory:
+1. Run the following from the project root folder:
 
     ```
-    go get -u github.com/golang/protobuf/protoc-gen-go
-    protoc --go_out=./gtfs/ --proto_path=.. ../gtfs-realtime.proto
+    docker build -t gtfs-golang -f golang/Dockerfile .
+    # -it to make sure docker run can be killed with ctrl-c
+    # -t uses TTY, which causes linux to include carriage returns, which are stripped using tr
```

Comment:
The -t flag isn't actually used in the command below?  Also, I don't see \r characters in my output file when I run the this command on my own machine.

Reply:
@bdferris-v2 what is the exact docker command that you're running?

Reply:
just checking - you've run this with `-t` and without `| tr -d '\r'`?

Reply:
Ah, apologies.  Not much of a Docker user.  I thought -it was a single flag, not -i + -t.  Also, I just did a plain run without -it, which admittedly does not add \r but I guess I see your point that the process could theoretically not be responsive to ctrl-c.  Resolving for now.

Reply:
no worries!